docs/source/sysadmin_guide/pbft_adding_removing_node.rst

```diff
@@ -0,0 +1,163 @@
+******************************
+Adding or Removing a PBFT Node
+******************************
+
+Membership of a Sawtooth PBFT network is controlled by the on-chain setting
+``sawtooth.consensus.pbft.members``. An administrator must update this setting
+when :ref:`adding a new node <adding-a-pbft-node-label>` or
+:ref:`removing an existing node <removing-a-pbft-node-label>`.
+
+All nodes in the network monitor this setting. When a node detects a change,
+it updates its local list of PBFT members to match the new list. When all member
+nodes have updated the setting, consensus resumes normal operation.
```

Comment:
This last sentence isn't really accurate; none of the nodes wait until other nodes update the setting, they just update their own then continue on.

Reply:
Removed that sentence.